src/main/kotlin/snyk/iac/IacSuggestionDescriptionPanel.kt

```diff
@@ -272,6 +276,27 @@
         return remediationPanel
     }
 
+    private fun addIssueReferences() {
+        val referenceRowCount = if (issue.references.isNotEmpty()) issue.references.size + 1 else 0
+        if (referenceRowCount > 0) {
+            val panel = JPanel()
+            panel.layout = GridLayoutManager(
+                referenceRowCount + 1,
+                1,
+                Insets(20, 0, 20, 0),
+                50,
+                -1
+            )
+
+            panel.add(boldLabel("References"), baseGridConstraints(row = 1))
+            issue.references.forEachIndexed { index, s ->
+                val label = labelProvider.createLinkLabel(URL(s), s)
```

Comment:
minor: not obvious why `row=7` and what `this` is here. Shell we move that addition to main panel init{} as it done for others panels (like remediationPanelWithTitle)?